crt_portal/static/sass/custom/form.scss

```diff
@@ -6,436 +6,441 @@
 form#details-edit-form,
 form#report-form,
 form#filters-form {
-  .question--header {
-    width: 100%; // IE11 fix
-    // margin-top: 0.5rem;
-    // margin-bottom: units($spacer-3x);
```

Comment:
This has a lot of tab/space related changes. I couldn't figure out what is the real change. If there are any real change, please revert space/tab changes and add just your changes.

Reply:
Took a bit to figure out but when I created this branch, 622-d, it was off of develop. These are changes from the active develop branch.